app/views/partials/workspaces/_list_card.html.erb

```diff
@@ -2,6 +2,7 @@
   <a
     href=<%= workspace_path(workspace) %>
     class="block flex w-ful gap-2 border border-grey py-4 px-4 rounded-lg bg-black hover:border-blue"
+    data-turbo="false"
```

Comment:
by  setting data-turbo to false, when a user clicks on a channel link, the browser load all the `workspace#show` and the `workspcae_channel` from actionable is connected